iotdb-core/node-commons/src/main/java/org/apache/iotdb/commons/auth/entity/PathPrivilege.java

```diff
@@ -30,33 +30,23 @@
 import java.io.DataOutputStream;
 import java.io.IOException;
 import java.nio.ByteBuffer;
-import java.util.Comparator;
 import java.util.HashSet;
 import java.util.Objects;
 import java.util.Set;
-import java.util.concurrent.atomic.AtomicInteger;
 
-/**
- * This class represents a privilege on a specific seriesPath. If the privilege is seriesPath-free,
- * the seriesPath will be null.I
- */
+/** This class represents a privilege on a specific seriesPath. */
 public class PathPrivilege {
   private static final Logger logger = LoggerFactory.getLogger(PathPrivilege.class);
-  private Set<Integer> privileges;
-  private PartialPath path;
 
-  /**
-   * This field records how many times this privilege is referenced during a life cycle (from being
-   * loaded to being discarded). When serialized to a file, this determines the order of
-   * serialization. The higher this values is, the sooner this privilege will be serialized. As a
-   * result, the hot privileges will be in the first place so that the hit time will decrease when
-   * being queried.
-   */
-  private AtomicInteger referenceCnt = new AtomicInteger(0);
+  // privilege capacity: read_data, write_data, read_schema, write_schema;
+  private static final Integer priCap = 4;
```

Comment:
Can we directly use Enum to init this field? Because in future, other  people add a new auth, he may forget to change here